ui/app/components/app/index.scss

```diff
@@ -57,3 +59,8 @@
 @import 'wallet-overview/index';
 @import '../ui/account-mismatch-warning/index';
 @import '../ui/icon-border/icon-border';
+@import 'connect-wallet/index';
+@import 'navigation/index';
+@import 'password-warning/index';
+@import 'custom';
+@import 'dark';
```

Comment:
also a question of curiosity -- I assume these are needed explicitly now because they were autoloaded by the old build task?

Reply:
That’s correct yes, so going forward any new UI component we just will need to add the import here.

Potential improvement in the future: I think the build task should be able to recursively scan for scss files in the ui directory and handle adding these for us